handshaker_test.go

```diff
@@ -51,6 +52,7 @@ func TestHandshaker(t *testing.T) {
 					cancelCli()
 				}
 			},
+			retransmitInterval: 100 * time.Millisecond,
```

Comment:
Is there a reason why it's 100? Should this maybe be a constant so the name can document its purpose?

Reply:
Added const. It just must be non-zero.